yt/frontends/gizmo/tests/test_outputs.py

```diff
@@ -75,3 +73,52 @@ def test_gizmo_mhd():
         f = ad[ptype, fmag + '_' + axis]
         assert str(f.units) == 'code_magnetic'
         assert f.shape == (409013,)
+
+
+@requires_ds(gmhd, big_data=True)
```

Comment:
I'm not sure if it's necessary to set `big_data=True` here.

Reply:
It doesn’t make any difference

Reply:
Actually both of the new tests you added aren't answer tests and shouldn't use `required_ds` at all. Instead use `requires_file`. That way they will run as part of the unit tests if anyone has the test dataset you're using in their `test_data_dir` path.

Reply:
Thanks! I didn't know this.